src/assets/js/modules/TodoItems.js

```diff
@@ -14,7 +16,56 @@ class TodoItems {
     displayTodos.display();
   }
 
-  static deletetodo() {}
+  static deletetodo(id) {
+    let todosArr = JSON.parse(localStorage.getItem('todos'));
+    todosArr = todosArr.filter((todo) => todo.index !== id);
+    todosArr = updateIndex(todosArr);
+    localStorage.setItem('todos', JSON.stringify(todosArr));
+    this.todoitems = todosArr;
+    displayTodos.display();
+  }
```

Comment:
- [ ] **Required:** You have done an amazing job. 👏 However, you are required to have a delete function aside from the `clear all completed ` button function. This functions to delete both completed or uncompleted single items and is fired upon clicking the trashcan icon in the video.

![Delete](https://user-images.githubusercontent.com/27709832/161163750-76790a81-94e2-4ba2-a733-7e27491a6126.gif)